packages/mui-docs/src/ComponentLinkHeader/ComponentLinkHeader.tsx

```diff
@@ -97,6 +98,26 @@ export function ComponentLinkHeader(props: ComponentLinkHeaderProps) {
           />
         </Tooltip>
       </li>
+      {headers.githubSource ? (
+        <li>
+          <Chip
+            clickable
+            role={undefined}
+            component="a"
+            size="small"
+            variant="outlined"
+            rel="nofollow"
+            href={`${process.env.SOURCE_CODE_REPO}/tree/${process.env.SOURCE_GITHUB_BRANCH}/${headers.githubSource}`}
```

Comment:
Why not do the same thing as with the demos? This link we break once we version the docs pages:

https://github.com/mui/material-ui/blob/4b31d50f9e1b5c30c6d20c49dd5796d39329944c/docs/src/modules/components/RichMarkdownElement.js#L132

Reply:
Was not aware of this part of the code

Reply:
Alright, then I assume we can move to do the same 😄